chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DownloadableResourceController.java

```diff
@@ -236,4 +286,11 @@ public void downloadSurveillance(@RequestParam(value = "type", required = false,
         LOGGER.info("Downloading " + downloadFile.getName());
         fileUtils.streamFileAsResponse(downloadFile, "text/csv", response);
     }
+
+    @RequestMapping(value = "/listings-with-svap",
+    method = RequestMethod.GET,
+    produces = "application/json; charset=utf-8")
+    public @ResponseBody List<Long> getListingIdsWithSvap() {
```

Comment:
Is this method used? Is the endpoint used? This seems like an odd method to have here